mapper/orm/src/main/java/org/hibernate/search/mapper/orm/model/impl/HibernateOrmBootstrapIntrospector.java

```diff
@@ -56,7 +57,11 @@
 	public static HibernateOrmBootstrapIntrospector create(Metadata metadata,
 			ReflectionManager ormReflectionManager,
 			ConfigurationPropertySource propertySource) {
-		Collection<PersistentClass> persistentClasses = metadata.getEntityBindings();
+		Collection<PersistentClass> persistentClasses = metadata.getEntityBindings()
+				.stream()
+				.filter( PersistentClass::hasPojoRepresentation )
+				.filter( persistentClass -> !"org.hibernate.envers.DefaultRevisionEntity".equals( persistentClass.getClassName() ) )
```

Comment:
I'm not sure why you need to explicitly exclude `DefaultRevisionEntity`? From what I can see it doesn't have the same issue as the other "non-pojo" persistent classes, i.e. it won't trigger an NPE, and it's not annotated with `@Indexed` so Hibernate Search should ignore it in the end (it will just collect metadata and then forget about it).